src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 600;
+}
+
+html {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.input {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-color: #fff;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  height: 66px;
+  font-size: 16px;
+  font-family: inherit;
+  background-position: 26px 25px;
+  padding-left: 62px;
+  width: calc(100% - 62px);
+}
+
+.input::placeholder {
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.input--small::placeholder {
+  font-size: 14px;
+  color: #3d4e61;
+}
+
+.input:hover {
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  padding-left: 62px;
+  width: calc(100% - 62px);
+  outline: none;
+}
+
+.input--small {
+  height: 38px;
+  font-size: 14px;
+  font-family: inherit;
+  background-size: 1.55%;
```

Comment:
![image](https://user-images.githubusercontent.com/97547824/211384324-8661b7c9-208a-4b89-bdce-fbe6beae68c3.png)
icon is too big